assets/js/html5shiv-3.7.3.min.js

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
wonder if this is required.. we don't have in prison staff hub for instance. 
Guess got to consider lowest supported browser.